### PR TITLE
Add MoneroTipper to the project

### PR DESCRIPTION
**Pull request by erciccione**
see #45

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Feel free to open an issue if you have suggestions/proposals.
 | [go-monero-rpc-client](https://github.com/monero-ecosystem/go-monero-rpc-client) | client for the Monero RPC (wallet and daemon) in golang | omani |
 | [monero-rpc-rs](https://github.com/monero-ecosystem/monero-rpc-rs) | RPC client written in Rust for Monero daemon and wallet | vorot93 |
 | [vanity-monero](https://github.com/monero-ecosystem/vanity-monero) | Generate vanity address for Monero | Equim-chan |
+| [MoneroTipper](https://github.com/monero-ecosystem/MoneroTipper) | non-custodial Monero tip bot for reddit | dginovker |
```
Comment (dginovker) [inline readme.md]: Can "non" and "reddit" be capitalized here?

File: readme.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ Feel free to open an issue if you have suggestions/proposals.
 | [go-monero-rpc-client](https://github.com/monero-ecosystem/go-monero-rpc-client) | client for the Monero RPC (wallet and daemon) in golang | omani |
```
Comment (dginovker) [inline readme.md]: For consistency, can "client" likewise be capitalized too?

**Comment by erciccione** [comment]
fixed :)

**Comment by dginovker** [comment]
Looks good to me :)

**Pull request merged**